LootLockerServerSDK/Source/LootLockerServerSDK/Public/LootLockerServerForBlueprints.h

```diff
@@ -1173,4 +1174,89 @@ class LOOTLOCKERSERVERSDK_API ULootLockerServerForBlueprints : public UObject
      */
     UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Character Progressions")
     static void DeleteProgressionForCharacter(int PlayerID, int CharacterID, const FString& ProgressionKey, const FLootLockerServerDeleteCharacterProgressionResponseBP& OnCompletedRequest);
+
+    //==================================================
+    // Instance Progressions https://ref.lootlocker.com/server-api/#asset-instance-progressions
+    //==================================================
+
+     /**
+     * Get a list of progressions for the specified instance
+     * The list will contain the first 20 items, if you want to get more items or not list items from somewhere else than the beginning; use GetPaginatedProgressionsForInstance
+     * https://ref.lootlocker.com/server-api/#get-asset-instance-progressions
+     *
+     * @param AssetInstanceId ID of the instance the specified progression belongs to
+     * @param OnCompletedRequest Delegate for handling the server response
+     */
+    UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | instance Progressions")
```

Comment:
```suggestion
    UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Instance Progressions")
```
Please capitalize Instance in all of these